docs/_docs/user-docs/recipe-headers.md

```diff
@@ -254,76 +254,7 @@ patient's identity was removed.
 The table below shows the full details of how multiple actions defined on the same
 field will interact with each other. In general the second action encountered on a field will supercede the first, however there are a few cases in which the actions are combined or the first supercedes the second.
 
-<table>
-    <tr>
-        <th colspan=2></th>
-        <th colspan=6>Second Action</th>
-    <tr>
-    <tr>
-        <th colspan=2></th>
-        <th>ADD</th>
-        <th>BLANK</th>
-        <th>JITTER</th>
-        <th>KEEP</th>
-        <th>REPLACE</th>
-        <th>REMOVE</th>
-    <tr>
-    <tr>
-        <th rowspan=6>First Action</th>
-        <th>ADD</th>
-        <td>ADD #2</td>
-        <td>BLANK</td>
-        <td>ADD + JITTER<hr/><i><b>Actions Combined</b></i></td>
-        <td>ADD<hr/><i><b>First Action Overrides Second</b></i></td>
-        <td>REPLACE</td>
-        <td>REMOVE</td>
-    </tr>
-    <tr>
-        <th>BLANK</th>
-        <td>ADD</td>
-        <td>BLANK</td>
-        <td>BLANK<hr/><i><b>First Action Overrides Second</b></i></td>
-        <td>KEEP</td>
-        <td>REPLACE</td>
-        <td>REMOVE</td>
-    </tr>
-    <tr>
-        <th>JITTER</th>
-        <td>ADD</td>
-        <td>BLANK</td>
-        <td>JITTER #1 + JITTER #2<hr/><i><b>Actions Combined</b></i></td>
-        <td>KEEP</td>
-        <td>REPLACE</td>
-        <td>JITTER<hr/><i><b>First Action Overrides Second</b></i></td>
-    </tr>
-    <tr>
-        <th>KEEP</th>
-        <td>ADD</td>
-        <td>KEEP<hr/><i><b>First Action Overrides Second</b></i></td>
-        <td>KEEP<hr/><i><b>First Action Overrides Second</b></i></td>
-        <td>KEEP<hr/><i><b>First Action Overrides Second</b></i></td>
-        <td>KEEP<hr/><i><b>First Action Overrides Second</b></i></td>
-        <td>KEEP<hr/><i><b>First Action Overrides Second</b></i></td>
-    </tr>
-    <tr>
-        <th>REPLACE</th>
-        <td>ADD</td>
-        <td>BLANK</td>
-        <td>REPLACE + JITTER<hr/><i><b>Actions Combined</b></i></td>
-        <td>KEEP</td>
-        <td>REPLACE #2</td>
-        <td>REPLACE<hr/><i><b>First Action Overrides Second</b></i></td>
-    </tr>
-    <tr>
-        <th>REMOVE</th>
-        <td>ADD</td>
-        <td>REMOVE<hr/><i><b>First Action Overrides Second</b></i></td>
-        <td>JITTER</td>
-        <td>KEEP</td>
-        <td>REPLACE</td>
-        <td>REMOVE</td>
-    </tr>
-</table>
+![Action Interaction Grid](/assets/img/interaction-grid.png)
```

Comment:
Did you render locally and it looks OK?

Reply:
I think I would use a relative path here - this might expect to be under pydicom.github.io and not deid/

Reply:
It did render locally, but looking at what's out there on pydicom.github.io, this would fail.  Just pushed the conversion to a relative path.